MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/offline/OfflineDownloadTest.kt

```diff
@@ -53,6 +54,7 @@ class OfflineDownloadTest : OfflineRegion.OfflineRegionObserver {
   }
 
   @Test(timeout = 60000)
+  @Ignore
```

Comment:
Do we ant to fix that issue?
If yes it could be a good idea to paste link to github issue here.